src/mysqloperator.tex

```diff
@@ -20,31 +20,31 @@ \section{High-level overview}
 \subsection{Functionalities}
 The MySQL Operator allows the user to create, delete, update, backup and restore MySQL clusters.
 
-When \textbf{creating} the cluster, the operator deploys a new instance of a replicated MySQL
```

Comment:
Maybe `textit` for the previously bolded parts? We're not using this decoration anywhere.